lambda/decode_function.py

```diff
@@ -0,0 +1,9 @@
+import boto3
+
+
+def object_body_to_str(bucket_name=str, object_key=str):
+    s3 = boto3.client('s3')

```

Comment:
The s3 client should only be initialized once, not every time you want to read a file, you should create a class for this method.

Reply:
Also, function names with the form x_to_y are usually mapping functions, where you pass a parameter of type x and map it to an object of type y. A better name for this function would just be `read_object_content`.